cluster/charts/library/templates/_recommended-labels.tpl

```diff
@@ -0,0 +1,9 @@
+{{/*
+Common labels
+*/}}
+{{- define "library.rook-ceph.labels" -}}
+app.kubernetes.io/part-of: rook-operator
```

Comment:
actually we should replace all of the `rook-operator` labels with `rook-ceph-operator`
```suggestion
app.kubernetes.io/part-of: rook-ceph-operator
```